### PR TITLE
Drone configuration for Moodle UI test

### PR DESCRIPTION
**Pull request by S-Panta**
This PR adds drone CI config to run moodle UI tests.
Related issue: https://github.com/owncloud/moodle-repository_ocis/issues/16

### CODE REVIEW

**Comment by amrita-shrestha** [comment]
@S-Panta your PR will be blocked by https://github.com/owncloud/moodle-repository_ocis/pull/34

File: .drone.star [hunk 1]
```diff
@@ -111,17 +156,212 @@ def release(ctx):
...
+            ],
+            "trigger": {
+                "ref": [
+                    "refs/pull/**",
+                ],
             },
         },
     ]
 
+def databaseService():
```
Comment (amrita-shrestha) [inline .drone.star:198]: adding this step to service maybe better option

File: .drone.star [hunk 2]
```diff
@@ -111,17 +156,212 @@ def release(ctx):
...
+    return [
+        {
+            "name":"behat-test",
+            "image":MOODLEHQ_APACHE,
+            "environment": MOODLE_ENV,
+            "commands": [
+                "update-ca-certificates",
+                "cd /var/www/html",
+                # run the test
+                # "vendor/bin/behat --config /var/www/behatdata/behatrun/behat/behat.yml repository/ocis/tests/behat/uploadFileToMoodle.feature",
```
[the remark this answers is not in the document]
Comment (S-Panta) [reply .drone.star:348]: Because test file isn't added and therefore this step will fail. commenting this out once ui test file get merged

**Comment by S-Panta** [comment]
Also, the test fails normally because the filepicker shows `undefined array key sortorder`  and filepicker doesn't work. That step should be added to the moodle repository code to make CI green for UI test. 
Also, see https://tracker.moodle.org/browse/MDL-81005

File: .drone.star [hunk 3]
```diff
@@ -111,17 +156,212 @@ def release(ctx):
...
+            "commands": [
+                "cd /usr/local/share/ca-certificates/",
+                "openssl req -x509  -newkey rsa:2048 -keyout ocis.pem -out ocis.crt -nodes -days 365 -subj '/CN=ocis'",
+                "openssl req -x509  -newkey rsa:2048 -keyout moodle.key -out moodle.crt -nodes -days 365 -subj '/CN=apache'",
+                "chmod -R 755 /usr/local/share/ca-certificates/",
+            ]
+        }
+    ]
+
+def apacheService():
```
[the remark this answers is not in the document]
Comment (S-Panta) [reply .drone.star:266]: it create environment to serve moodle like running apache on https rather than http

**Comment by S-Panta** [comment]
Drone CI will pass only after https://github.com/owncloud/moodle-repository_ocis/pull/87.

**Pull request merged**